src/climate_data/data.py

```diff
@@ -154,15 +154,26 @@ def load_scenario_inclusion_metadata(self) -> pd.DataFrame:
     def daily_results_path(self, scenario: str, variable: str, year: int | str) -> Path:
         return self.daily_results / scenario / variable / f"{year}.nc"
 
+    def daily_results_path_with_draws(
+        self, scenario: str, variable: str, draw: int | str, year: int | str
+    ) -> Path:
+        return self.daily_results / scenario / variable / f"{year}_{draw}.nc"
+
     def save_daily_results(
         self,
         results_ds: xr.Dataset,
         scenario: str,
         variable: str,
+        draw: int | str | None,
         year: int | str,
+        provenance_attribute: str | None,
```

Comment:
What is this?

Reply:
I write this as a global attribute lines 187-190.  There is no mechanism (that I saw) for knowing which source-variant a given draw is generated from.  Also was curious about how to add netcdf attributes to the output files.